src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -100,5 +132,33 @@ public void AddTagHelperAttribute([NotNull] string name, object value)
         {
             AllAttributes.Add(name, value);
         }
+
+        /// <summary>
+        /// Executes the child content asynchronously.
+        /// </summary>
+        /// <returns>A task that indicates when child content execution has completed.</returns>
```

Comment:
"renders" doesn't make sense here since ideally we're just executing what's inside (doesn't need to render anything).